vnext/Microsoft.ReactNative/Base/CoreNativeModules.cpp

```diff
@@ -35,6 +40,11 @@ bool HasPackageIdentity() noexcept {
   return hasPackageIdentity;
 }
 
+ReactPropertyId<bool> HttpUseMonolithicModuleProperty() noexcept {
```

Comment:
@asklar Please review this usage of the React Context's property bag.

Reply:
We've used the convention of namespace="ReactNative.xyz", prop name="Something".
I see that there are a few modules that diverged from this:
- http module
- blob module
- web socket module

all the other ones follow the existing convention. It would be good to stick to the convention for this module (it's fine to fix up the other ones in a separate PR).

So for this one I think we'd say:
```cpp
  static ReactPropertyId<bool> propId{L"ReactNative.HttpModul", L"UseMonolithicModule"};
```

similar in the other ones

Reply:
1) also I'm not super clear on what UseMonolithicModule means, consider a more descriptive name.

2) Is the idea that CreateHttpModule/"Networking" provides blob/filereader, or equivalent functionality, and then we override it if the "UseMonolithicModule" property isn't set?

3) under which conditions would you choose to use UseMonolithicModule or not? is there a tradeoff/benefit of setting it?

Reply:
The "monolithic module" is just the previous HTTP module implementation, which shoved all the business logic into the CxxModule subclass.

We are still giving an option to use that module in case of any issues with the current one, but we'll be most likely just deleting it soon.
This older module is not compatible with Blob support.